Lib/test/test_urllib2.py

```diff
@@ -1,5 +1,5 @@
 import unittest
-from test import test_support
+from test import support
```

Comment:
fwiw moving from test_support to support isn't directly related to this PR.  i don't mind myself.

Reply:
reverted